tests/functional/spec/features/calculated_summary.spec.js

```diff
@@ -4,7 +4,4 @@ describe("Feature: Calculated Summary", () => {
   describe("Given I have a Calculated Summary", () => {
```

Comment:
Here as well we still need to test the old and new calculated formats for the time being 👍

Reply:
Reverted this test file after bringing back the schema